src/store/ChartStore.js

```diff
@@ -231,25 +231,67 @@ class ChartStore {
             }
 
             const onLayoutDataReady = () => {
-                this.restoreLayout(stxx, layoutData);
-
-                this.setActiveSymbols(active_symbols);
-
-                if (initialSymbol && !(layoutData && layoutData.symbols)) {
-                    this.changeSymbol(initialSymbol);
-                } else if (stxx.chart.symbol) {
-                    this.currentActiveSymbol = stxx.chart.symbolObject;
-                    stxx.chart.yAxis.decimalPlaces = stxx.chart.symbolObject.decimal_places;
-                    this.categorizedSymbols = this.categorizeActiveSymbols();
-                    if (onSymbolChange) { onSymbolChange(this.currentActiveSymbol); }
-                } else {
-                    this.changeSymbol(this.defaultSymbol);
-                }
-
+                const today_utc = Date.UTC(
+                        (new Date()).getUTCFullYear(),
+                        (new Date()).getUTCMonth(),
+                        (new Date()).getUTCDate(),
+                        (new Date()).getHours(),
+                        (new Date()).getMinutes(),
+                        (new Date()).getSeconds(),
+                    ),
+                    timeToEpochGMT = (time_stirng) => {
```

Comment:
This can be simplified to:
```js
const timeToEpochGMT = (time_stirng) => {
    const dateStr = new Date().toISOString().substring(0, 11);
    return new Date(`${dateStr}${time_stirng}Z`);
};
```

Also that `time_stirng` typo.